airflow/cli/cli_parser.py

```diff
@@ -778,6 +782,12 @@ class GroupCommand(NamedTuple):
         func=lazy_load_command('airflow.cli.commands.dag_command.dag_list_dags'),
         args=(ARG_SUBDIR, ARG_OUTPUT),
     ),
+    ActionCommand(
+        name='generate_yaml',
+        help="Generate YAML files for all tasks in DAG",
+        func=lazy_load_command('airflow.cli.commands.dag_command.generate_pod_yaml'),
+        args=(ARG_SUBDIR, ARG_DAG_ID_OPT,ARG_OUTPUT_PATH),
```

Comment:
Is the DAG ID really optional?

Reply:
might be an error, should not be optional

Reply:
How do I make this not optional?